tests/src/test/scala/scalaz/EitherTTest.scala

```diff
@@ -81,6 +81,7 @@ object EitherTTest extends SpecLite {
     def bifunctor[F[_] : Traverse] = Bifunctor[EitherT[F, ?, ?]]
     def bifoldable[F[_] : Traverse] = Bifoldable[EitherT[F, ?, ?]]
     def monadError[F[_] : Monad, A] = MonadError[EitherT[F, A, ?], A]
+    def nondeterminism[F[_] : Nondeterminism, A] = Nondeterminism[EitherT[F, A, ?]]
```

Comment:
Will also need tests to determine that the instances of superclasses of `Nondeterminism` are still reachable in the context of `F[_]: Nondeterminism`.